.github/workflows/CI.yml

```diff
@@ -539,6 +539,7 @@ jobs:
           restore-keys: ${{ runner.os }}-venv-poetry_${{steps.get-poetry-version.outputs.poetry-version}}
       - name: Install libxml2-dev and tesseract-ocr
         run: |
+          poetry env use 3.10
```

Comment:
Is this really needed? Don't we specify the python version above?

Reply:
No but why does a python test workflow fail 😢

Reply:
Hmm I can try removing the cache and see if that helps.